internal/otelcollector/config/common/constants.go

```diff
@@ -33,57 +38,95 @@ var InstrumentationScope = map[InputSourceType]string{
 
 const (
-	// Receivers
-	ComponentIDOTLPReceiver    = "otlp"
-	ComponentIDFileLogReceiver = "filelog/%s" // dynamically filled with pipeline name
-
-	// Processors
-	ComponentIDMemoryLimiterProcessor           = "memory_limiter"
-	ComponentIDBatchProcessor                   = "batch"
-	ComponentIDK8sAttributesProcessor           = "k8sattributes"
-	ComponentIDServiceEnrichmentProcessor       = "service_enrichment"
-	ComponentIDIstioEnrichmentProcessor         = "istio_enrichment"
-	ComponentIDIstioNoiseFilterProcessor        = "istio_noise_filter"
-	ComponentIDSetObservedTimeIfZeroProcessor   = "transform/set-observed-time-if-zero"
-	ComponentIDSetInstrumentationScopeProcessor = "transform/set-instrumentation-scope-runtime"
-	ComponentIDUserDefinedTransformProcessor    = "transform/user-defined-%s" // dynamically filled with pipeline name
-	ComponentIDDropIfInputSourceOTLPProcessor   = "filter/drop-if-input-source-otlp"
-	ComponentIDNamespaceFilterProcessor         = "filter/%s-filter-by-namespace"
-	ComponentIDInsertClusterAttributesProcessor = "resource/insert-cluster-attributes"
-	ComponentIDDropKymaAttributesProcessor      = "resource/drop-kyma-attributes"
-
-	// Exporters
+	// ================================================================================
+	// RECEIVERS
+	// ================================================================================
+
+	ComponentIDOTLPReceiver      = "otlp"
+	ComponentIDFileLogReceiver   = "filelog/%s" // dynamically filled with pipeline name
+	ComponentIDKymaStatsReceiver = "kymastats"
+
+	// ================================================================================
+	// PROCESSORS
+	// ================================================================================
+
+	// Common Processors
+	ComponentIDBatchProcessor                          = "batch"
+	ComponentIDMemoryLimiterProcessor                  = "memory_limiter"
+	ComponentIDK8sAttributesProcessor                  = "k8sattributes"
+	ComponentIDServiceEnrichmentProcessor              = "service_enrichment"
+	ComponentIDIstioNoiseFilterProcessor               = "istio_noise_filter"
+	ComponentIDSetInstrumentationScopeKymaProcessor    = "transform/set-instrumentation-scope-kyma"
+	ComponentIDSetInstrumentationScopeRuntimeProcessor = "transform/set-instrumentation-scope-runtime"
+	ComponentIDUserDefinedTransformProcessor           = "transform/user-defined-%s" // dynamically filled with pipeline name
+	ComponentIDInsertClusterAttributesProcessor        = "resource/insert-cluster-attributes"
+	ComponentIDDropKymaAttributesProcessor             = "resource/drop-kyma-attributes"
+
+	// Log-Specific Processors
+	ComponentIDNamespaceFilterProcessor       = "filter/%s-filter-by-namespace" // dynamically filled with pipeline name and input source
+	ComponentIDSetObservedTimeIfZeroProcessor = "transform/set-observed-time-if-zero"
+	ComponentIDIstioEnrichmentProcessor       = "istio_enrichment"
```

Comment:
also, is the `ComponentIDIstioEnrichmentProcessor` specific to logpipelines only?

Reply:
Yes, it's only purpose is to enrich Istio access logs. The name is too generic and, thus, confusing, in my opinion.